linux/utils/add_official_online_repo.yml

```diff
@@ -200,6 +200,15 @@
         option_value: 1
       with_items: "{{ photon_online_repos }}"
 
+    - name: "Disable VMware Photon OS online repository - photon.repo"
```

Comment:
Is the repo photon.repo enabled by defalut for Photon OS 5?
If not, no need to disable it here, since from line 191, only two repos  photon-release and photon-updates will be enabled when photon OS >=4.

Reply:
Yes, it is enabled by default.